vars/commonPipelineEnvironment.groovy

```diff
@@ -25,11 +41,35 @@ class commonPipelineEnvironment implements Serializable {
             return configProperties[property]
     }
 
+    def setInfluxCustomData(Map myData = [:]) {
+        influxCustomData = myData
+    }
+
+    def getInfluxCustomData() {
+        return influxCustomData
+    }
+
+    def setInfluxCustomDataMap(Map myData = [:]) {
```

Comment:
Two questions:
1.) Does it make sense to provide ```myData``` with the empty map as default? It is strongly required to have an entry ```pipeline_data```in that map. This entry is not provided by the default map. Without this entry ```setPipelineMeasurement```and the corresponding getter will fail.
2.) Does it make sense at all to provide a setter which replaces the original map? Also here: there is a rule which needs to be observed when providing a new map (```pipeline_data``` must be present). Is there any use case where it is desirable to drop old values. In case there is this use case: what about some kind of a ```clear()``` which resets the map into a valid initial state?